src/main/java/com/icthh/xm/ms/balance/domain/Metadata.java

```diff
@@ -0,0 +1,71 @@
+package com.icthh.xm.ms.balance.domain;
+
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.unmodifiableMap;
+
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Transient;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.SneakyThrows;
+
+@Data
+@Embeddable
+@EqualsAndHashCode
+public class Metadata implements Serializable {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper()
+        .configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true)
+        .configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
+
+    @Transient
+    private Map<String, String> metadata = null;
```

Comment:
@sergeysenja1992 maybe it worth to make metadata with type Map<String, Objetc> to be able to store Numbers as well. What do you think?

Reply:
no 
it future it will be table
it's like additional labels, not complex object